DBZ-2501 Unify doc for each connector to have consistent event descriptions

TovaCohen opened the pull request:
This is for [DBZ-2501](https://issues.redhat.com/browse/DBZ-2501).

The updated doc for the descriptions of the MySQL events was merged in https://github.com/debezium/debezium/pull/1747. 
For DBZ-2501, the work was to use the updated MySQL content as a model for the event descriptions in the doc for each of the other connectors. 

This PR updates the event description doc for each connector. There is a separate commit for each connector.

Naros commented on documentation/modules/ROOT/pages/connectors/mongodb.adoc:
```diff
@@ -446,46 +589,71 @@ The value of an _update_ change event on this collection will actually have the
 ====
-Update events in MongoDB's oplog do not have the _before_ or _after_ states of the changed document, so there's no way for the connector to provide this information.
-However, because _create_ or _read_ events _do_ contain the starting state, downstream consumers of the stream can actually fully-reconstruct the state by keeping the latest state for each document and applying each event to that state. {prodname} connector's are not able to keep such state, so it is not able to do this.
+In MongoDB's oplog, _update_ events do not contain the _before_ or _after_ states of the changed document. Consequently, it is not possible for a {prodname} connector to provide this information. However, MongoDB's oplog contains a document's starting state in _create_ and _read_ events. Downstream consumers of the stream can reconstruct document state by keeping the latest state for each documen ...
```
I think the following is slightly misleading:
```
MongoDB's oplog contains a document's starting state in _create_ and _read_ events.
```
For _read_ events, we don't consult the oplog but rather we stream the contents of the actual collection directly, just like we stream rows directly from the base tables for relational databases.  I think it might be sufficient to reword the above to something like:
```
However, the {prodname} connector can provide starting state for _create_ and _read_ events.
```

TovaCohen replied:
@Naros - can or always does? Is this correct: 

However, a {prodname} connector provides a document's starting state in _create_ and _read_ events.

TovaCohen replied:
I just committed an update with the wording I suggested. Let me know if something needs to change.

Naros replied:
LGTM, thanks!

Naros reviewed (changes requested):
Overall this LGTM.  I just found one tiny nit-pick.

TovaCohen commented:
To fix a rendering problem downstream, I just updated the doc for each connector, in its section that describes create events. I had renumber the callouts so there were no duplicates. I updated the description tables to reflect the new numbers. Also, I put back the comment slashes on the lines that have callouts in the event examples. I had removed them in an effort to fix the downstream rendering problem. But that didn't work. And the comment symbols in the event representation make it easier to copy and paste the content. I believe this is ready to merge. Let me know if not.

Naros commented:
Applied to master & 1.2
